### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by quangpt-0159**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# collaboration-test
\ No newline at end of file
+# collaboration-test
+
+Test branch rule
```
Comment (quang-pth) [inline README.md:3]: Nice job bro

Comment (quang-pth) [inline README.md:3]: He

**Review by quang-pth — COMMENTED** [review]
EE

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# collaboration-test
```
Comment (quang-pth) [inline README.md:1]: You should not delete this line

Comment (quangpt-0159) [reply README.md:1]: Ok

**Review by quang-pth — APPROVED** [review]

**Pull request merged**